modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/dnd_table/Cell.tsx

```diff
@@ -50,9 +50,28 @@ const Cell = ({
 	const clientXRef = useRef({current: null});
 
 	useEffect(() => {
+		const CELL_WIDTH = {
+			LARGE: 82,
+			SMALL: 70,
+		};
+
 		if (columnName && heading && !isFixed && cellRef.current) {
 			const boundingClientRect = cellRef.current.getBoundingClientRect();
 
+			if (columnName !== 'item-actions') {
+				if (boundingClientRect.width < CELL_WIDTH.SMALL) {
+					const fontSize = getComputedStyle(cellRef.current).fontSize;
+
+					boundingClientRect.width += parseFloat(fontSize) * 4;
+				}
+
+				if (boundingClientRect.width < CELL_WIDTH.LARGE) {
+					const fontSize = getComputedStyle(cellRef.current).fontSize;
+
+					boundingClientRect.width += parseFloat(fontSize) * 2;
+				}
```

Comment:
Is it intentional that width for small size is increased by `fontSize * 6`? Is `>=` missing here?

Reply:
Hello @markocikos,

I added <= so that all cells less than or equal to 70px are not increased by  `parseFloat(fontSize) * 6;`. I made this change since I found other cells that are still being cut such as the ERC in the following:

![Screenshot from 2024-07-29 14-04-50](https://github.com/user-attachments/assets/97abc39a-fbbf-402c-bed3-34b8611331ec)


I also increased the the medium cells, between 70 and 82, by `parseFloat(fontSize) * 4;`.

Please let me know if there are any questions or comments about this.
Thank you!

Reply:
@fortunatomaldonado  Honestly, I really don't like that we fine-tune thresholds based on one use case. They will just fail on another. And I don't like obfuscation caused by incrementing the value twice.

Can we just set flat minimum width? Something like
```js
if (boundingClientRect.width < 100) {
	boundingClientRect.width = 100;
}
```

Reply:
@markocikos I agree with you on this. 

I believe we should have a higher minimum width like 140. I have some examples of the difference between 100 and 140. 

If there are any further cells that get cut, then that is when the user should resize the cell.
Let me know what you think. 

Thank you for all the help with this!

100px
![100px-Place-Order](https://github.com/user-attachments/assets/baf6f0f1-7a09-40db-81ae-272bbbd78bba)


140px
![140-Place-Orders](https://github.com/user-attachments/assets/16ada727-9317-48d5-85c2-e22816f66890)


100px
![100-order](https://github.com/user-attachments/assets/bcdd8f99-45c0-4675-abdc-be73f1654057)


140px
![140-order](https://github.com/user-attachments/assets/15e0bdc2-69e5-49e3-835c-bc5c66746f62)